src/OrbitQt/CallTreeWidget.cpp

```diff
@@ -114,6 +119,20 @@ void CallTreeWidget::SetCallTreeView(std::unique_ptr<CallTreeView> call_tree_vie
   ResizeColumnsIfNecessary();
 }
 
+void CallTreeWidget::SetInspection(std::unique_ptr<CallTreeView> call_tree_view) {
+  ui_->inspectionNoticeWidget->show();
+  inspection_model_ = std::make_unique<CallTreeViewItemModel>(std::move(call_tree_view));
```

Comment:
A note that this works for both the top-down and the bottom-up view because `SetTopDownView` or `SetBottomUpView` have already created and set their `HideValuesForTopDownProxyModel`/`HideValuesForBottomUpProxyModel` and potentially called `ui_->callTreeTreeView->hideColumn`.
Probably not much to do. Maybe it could be clearer if the definition of this method was together with `SetTopDownView`/`SetBottomUpView`, which are public like this one, while `SetCallTreeView` is an internal method.

Reply:
Good point; I moved `SetInspection` & `ClearInspection` to be after `SetTopDownView` and `SetBottomUpView` and added a comment that they must be called after those methods. I also added a CHECK for hide_values_proxy_model_ which should make it more clear for anyone developing where/why it failed since we dereference that and it should be getting set during those `Set*View` methods.